ann_benchmarks/__init__.py

```diff
@@ -213,18 +213,28 @@ def __init__(self, metric, P):
         self._metric = metric
 
     def fit(self, X):
+        os.environ['OMP_THREAD_LIMIT'] = '40'
```

Comment:
Yeah i know it's only for index construction. I think it's fine to use multiple threads for index construction. Maybe I'll add support for that in Annoy too